rainbow-me/rainbow
# Fix password field focus

Fixes RNBW-4079
Figma link (if any):

## What changed (plus any additional context for devs)

The password input is wrapped with touchable, which was causing problems on some phones when you could not actually focus the input.

I removed the touchable wrapper since it was basically doing nothing.


## Screen recordings / screenshots


https://user-images.githubusercontent.com/7809008/185788017-6f98983d-cde9-4253-94ed-d3f11e45e559.mov



## What to test


- Create a wallet, and try to backup it with google drive
- The first input should autofocus, but if it's not - you should be able to focus it
- If you hide the keyboard, the first input should be focusable too
- when you type a valid password, the second input should be focusable to
- Enter key on the keyboard should focus second input when the password is valid


## Final checklist
